.circleci/config.yml

```diff
@@ -26,18 +26,14 @@ jobs:
       - run:
           name: test connection via SQL CMD
           command: sqlcmd -S 'localhost,1433' -U sa -P 5atyaNadella -Q 'create database blog'
-      - python/install-packages:
-          pkg-manager: pip
       - run:
           name: Test adapter on SQL Server against dbt-adapter-tests
-          command: tox -e integration-sqlserver
+          command: tox -- -v test/integration/sqlserver.dbtspec
   connection-sqlserver:
     <<: *sqlserver
     steps:
       - run: *install-git
       - checkout
-      - python/install-packages:
```

Comment:
not needed since this is what tox does for you

Reply:
@JCZuurmond this why the steps are currently failing. Somehow `dbt==0.18.1` is getting installed. perhaps the Circle CI python orb auto creates `virtualenvs`? Once we get this resolved this PR is good to go! Thanks again for all the work

```
#!/bin/bash -eo pipefail
cd test/integration
dbt compile --target sqlserver_local_userpass
```
```
Running with dbt=0.18.1
Encountered an error:
cannot import name 'AdapterResponse' from 'dbt.contracts.connection' (/usr/local/lib/python3.7/site-packages/dbt/contracts/connection.py)
Exited with code exit status 2
```